packages/dart/npt_flutter/lib/features/profile_list/bloc/profile_list_bloc.dart

```diff
@@ -36,8 +41,32 @@ class ProfileListBloc extends LoggingBloc<ProfileListEvent, ProfileListState> {
       emit(const ProfileListFailedLoad());
       return;
     }
-
     emit(ProfileListLoaded(profiles: profiles));
+
+    // Create ProfileBlocs
+    final profileBlocs = <ProfileBloc>[];
+    for (final uuid in profiles) {
+      final bloc = _profileCacheCubit.getProfileBloc(uuid);
+      profileBlocs.add(bloc);
+    }
+
+    // Wait for all ProfileBlocs to load their data
+    await Future.wait(
+      profileBlocs.map((bloc) async {
+        // Wait until the bloc is in a loaded state
+        await bloc.stream.firstWhere(
+          (state) => state is ProfileLoadedState || state is ProfileFailedLoad,
+          orElse: () => bloc.state,
+        );
+      }),
+    );
+
+    add(
+      const ProfileListSortEvent(
+        sortColumn: SortColumn.profileName,
+        inverseSortOrder: false,
+      ),
+    );
```

Comment:
Tigger that sorts the column by profile name in ascending order when the profiles first loads